pages/common/cargo-check.md

```diff
@@ -0,0 +1,24 @@
+# cargo check
+
+> Check a local package and all of its dependencies for errors.
+> More information: <https://doc.rust-lang.org/cargo/commands/cargo-check.html>.
+
+- Check the current package:
+
+`cargo check`
+
+- Check all tests:
+
+`cargo check --tests`
+
+- Check the integration tests in `tests/integration_test1.rs`:
+
+`cargo check --test integration_test1`
+
+- Check the current package with the features "feature1" and "feature2":
+
+`cargo check --features feature1,feature2`
```

Comment:
Wouldn't this work?

```suggestion
- Check the current package with the features `feature1` and `feature2`:

`cargo check --features {{feature1,feature2}}`
```

Since the example seems to be only for 2 features. Or else we can update the description to

```suggestion
- Check the current package with multiple features:

`cargo check --features {{feature1,feature2,...}}`
```

Reply:
I think the first version of @kbdharun is nice.